docs/stdlib/constraints.rst

```diff
@@ -47,9 +46,7 @@ Constraints
 
 .. eql:constraint:: std::max_value(max: anytype)
 
-    Specifies the maximum value for the subject.
-
-    Example:
+    Specifies the maximum allowed value of :eql:type:`anytype`:
```

Comment:
```suggestion
    Specifies the maximum allowed value.
    
    Example:
```

Similar to above.

Reply:
Do we need to have a separate paragraph with "Example?" I personally think it's a little off, the colon in my mind represents us exemplifying the action based off of our description.